### ISSUE TITLE
Messy looks of documentation 

### ISSUE THREAD

**Issue by pangyyen** [pangyyen/pe#12]
The documentation of implementation lacks titles and subtitles, making it hard to follow through. 

![Screenshot 2023-11-17 at 5.25.28 PM.png](https://raw.githubusercontent.com/pangyyen/pe/main/files/680e8e8b-922c-48a1-bc43-ca027d1e08e8.png)

**Comment by nus-pe-script** [comment]
# Team's Response
Documentation has spacings and punctuation. Additionally, all parts are consistent and easy to follow. In future iterations, we might add titles and subtitles to provide greater accessibility.
# Items for the Tester to Verify
## :question: Issue response

Team chose [`response.NotInScope`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your explanation]

<catcher-end-of-segment><hr>
